README.md

```diff
@@ -160,7 +160,7 @@ Table of Contents
   * [chiselapp.com](http://chiselapp.com/) — Unlimited public and private Fossil repositories
   * [codebasehq.com](https://www.codebasehq.com/) — One free project with 100 MB space and 2 users
   * [Codeberg.org](https://codeberg.org/) - Unlimited public and private Git repos for free and open-source projects. Static website hosting with [Codeberg Pages](https://codeberg.page/).
-  * [CodeFever](https://codefever.cn) — Unlimited private repositories. Fully open source and lightweight desigin. Powered by PGYER Team.
+  * [CodeFever](https://github.com/PGYER/codefever) — Unlimited private repositories. Fully open source and lightweight desigin. Powered by PGYER Team.
```

Comment:
The github project does not tell me about the pricing plans or what is available for free in the feature sets provided by a Software As A Service offering.

Reply:
This is a completely open source project and is mentioned among his features as being completely free.

Reply:
This list is not for self hosted open source software